app/javascript/components/widgets/weather/current-weather.jsx

```diff
@@ -49,6 +54,28 @@ export const AnticipatedPrecipitation = ({ weather }) => {
   if (precip.length === 0) {
     return '';
   }
+  let precipitationType = 'precipitation';
+  if (
+    weather.hourly[0].weather.id >= 200 &&
```

Comment:
Might be a little less verbose to do something like `const weatherId = weather.hourly[0].weather.id`

Reply:
introduced variable 👍🏻